### ISSUE TITLE
[RFE] entries in sshd Match blocks indented

### ISSUE THREAD

**Issue by jcpunk** [hercules-team/augeas#286]
To aid human readers of the sshd_config, can entries augeas adds to a Match section be indented in some way?

Currently match lines added by augeas may not be easily noticed by humans, for example:
```
    Match LocalAddress 172.16.0.0/16
    AllowAgentForwarding yes
    AllowTcpForwarding yes
    X11Forwarding yes
    PermitRootLogin without-password
    Match LocalAddress 127.0.0.1
    PermitRootLogin yes
```

Where this would be much easier for humans to review:
```
    Match LocalAddress 172.16.0.0/16
     AllowAgentForwarding yes
     AllowTcpForwarding yes
     X11Forwarding yes
     PermitRootLogin without-password
    Match LocalAddress 127.0.0.1
     PermitRootLogin yes
```

**Comment by raphink** [comment]
Which version of Augeas are you using?

719c40e6 introduced a default indentation of two spaces for Match entries and was released as part of Augeas 1.3.0

**Comment by jcpunk** [comment]
I tested with augeas-1.1.0-17.el7, that explains the discrepancy.
